systemd/docker-compose.yaml

```diff
@@ -0,0 +1,82 @@
+# An example setup for a distributed sccache cluster that uses
+# - a single scheduler (10.0.0.10)
```

Comment:
I am not sure which private IP range is the most uncommon to use.

Reply:
I'm impartial, `10.0.0.0/24` might be somewhat popular. Do you want me to use some uncommon addresses like `10.198.37.0/24` or something?

Reply:
Yeah, I'd be in favor of using a less common subnet to avoid collision for the common case.

Reply:
Done in https://github.com/paritytech/sccache/pull/77/commits/561e171abfaa58424c01e7a24041e4f0441d6f3c (part of #77)